lib/webmock/http_lib_adapters/http_rb/response.rb

```diff
@@ -14,9 +14,17 @@ def self.from_webmock(webmock_response, request_signature = nil)
       status  = Status.new(webmock_response.status.first)
       headers = webmock_response.headers || {}
       body    = Body.new Streamer.new webmock_response.body
-      uri     = URI request_signature.uri.to_s if request_signature
+      uri     = URI.coerce request_signature.uri.to_s if request_signature
```

Comment:
Oh. I don't know what I was thinking of when pushed that. Thanks for catching this up.